### ISSUE TITLE
Step by step guide doesn't work with webpack 4

### ISSUE THREAD

**Issue by nicolls1** [django-webpack/django-webpack-loader#204]
https://owais.lone.pw/blog/webpack-plus-reactjs-and-django/

For one 'loaders' is changed to 'rules' 

Then:

```
$ ./node_modules/.bin/webpack --config webpack.config.js
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.resolve has an unknown property 'modulesDirectories'. These properties are valid:
   object { alias?, aliasFields?, cachePredicate?, cacheWithContext?, concord?, descriptionFiles?, enforceExtension?, enforceModuleExtension?, extensions?, fileSystem?, mainFields?, mainFiles?, moduleExtensions?, modules?, plugins?, resolver?, symlinks?, unsafeCache?, useSyncFileSystemCalls? }
   -> Options for the resolver
```

I'm not familiar with webpack/babel and an updated guide that just works would be appreciated!

**Comment by khadegd** [comment]
Hey @nicolls1, will this help? - https://github.com/khadegd/django-webpack-starter

**Comment by mightym** [comment]
@khadegd thanks for you example. I was happy that its running with webpack 4. But lately I realized that hmr is actually not working. When I run your project and change eg something in the `app.scss` I was happy as I saw the page on django reloading and my changes where there. BUT when looking closer what actually happens is: 

its trying to access the `hot-update.json` 
```
"GET / HTTP/1.1" 200 1566
Not Found: /4d7d3642bf922fb5b04c.hot-update.json
GET /4d7d3642bf922fb5b04c.hot-update.json HTTP/1.1" 404 2157
```
every time you make changes to your js/scss. Which causes the browser to reload.
